18F/identity-design-system
# Simplify and generalize navigation

## 🛠 Summary of changes

Updates how side navigation is generated within the local documentation site, to avoid tying it specifically to components. 

This is part of a series of changes looking to introduce new sections for "Utilities" in addition to components, which will need to have its own side navigation.

## 📜 Testing Plan

The only meaningful change is that visiting the root URL will redirect automatically to the first component.

1. `npm start`
2. Visit http://localhost:4000/
3. Observe that you are redirected to http://localhost:4000/accordions/
